microsetta_public_api/models/_factory.py

```diff
@@ -0,0 +1,88 @@
+import biom
```

Comment:
id just delete this file right now tbh. Let's let the interface guide all this

Reply:
https://github.com/biocore/microsetta-public-api/pull/1/commits/9cd4cf9b0caa095b0ff09528dc4fd0f8a19989ae